### PR TITLE
Zeid - cordova hookup

### PR DESCRIPTION
**Pull request by zedin27**
This PR can affect the master's `README.md`. Review first

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+## Authors
+
+- [Jose Ramón Aleman](https://github.com/jraleman)
+- [Daniel Gonzales]()
+- [Gerardo Malpartida](https://github.com/regien)
+- [Felipe Romo](https://github.com/felromo)
+- [Zeid Tisnes](https://github.com/zedin27)
+
+## Resources
+* [shitgoeshere (Cordova Apache)](https://cordova.apache.org/docs/en/latest/guide/cli/index.html)
```
Comment (jraleman) [inline README.md]: shitgoeshere?

File: README.md [hunk 2]
```diff
@@ -1 +1,49 @@
...
+* [moreshitgoeshere (Platform building)](https://cordova.apache.org/docs/en/latest/reference/cordova-cli/index.html#cordova-platform-command)
```
Comment (jraleman) [inline README.md]: moreshitgoeshere?

**Comment by jraleman** [comment]
Also, move `narwell-cordova/` to `src/cordova/`

File: README.md (hunk 2 above)
Comment (zedin27) [reply README.md]: hahahaha I was bored and i didnt know what to name my links lmao

Comment (regien) [reply README.md]: anyways, read the comments of ramon and move `narwell-cordova` in `src/cordova`

**Pull request closed without merging**